Automattic/wp-calypso
# Site Settings: hook, add and introduce Jetpack Confirm Disconnection site

This PR part of the final `Confirm Disconnection` site of the Jetpack Disconnect Flow.

Included:
- introducing the `Confirm Disconnection` site, hooking it up and adding route definitions.
- adding and loading the style 
- landing layout of the page
- adding the dev feature flag

**To test:**

Checkout this branch, or use calypso.live
    https://calypso.live/?branch=add/site-settings-confirm-disconnection

For Jetpack sites (incl. the dev mode):

- verify that the start of the site is visible at /settings/disconnect-site/:site/confirm, where :site is the Jetpack site.

For non-Jetpack and Atomic sites,

- verify immediate redirection to `/settings/general`

With empty Redux state we should see a placeholder prior to rendering.

**Notes:**
- For the time being, the `Placeholder` used is not specific to the Disconnect Flow. It is developed and reviewed elsewhere and will be updated here once complete.
- `Back` button (top left page content view) will be included in the follow-up PR

Visuals:
![screen shot 2017-09-19 at 00 21 47](https://user-images.githubusercontent.com/13561163/30569053-78636568-9cd0-11e7-9021-de982e5df8fa.png)

Final result:
![screen shot 2017-09-13 at 20 30 59](https://user-images.githubusercontent.com/13561163/30438841-2913483e-996a-11e7-99fc-17755fad9e74.png)
